lib/danger/danger_core/plugins/dangerfile_bitbucket_server_plugin.rb

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+require 'danger/plugin_support/plugin'
+
+module Danger
+  class DangerfileBitbucketServerPlugin < Plugin
+    def self.new(dangerfile)
+      return nil if dangerfile.env.request_source.class != Danger::RequestSources::BitbucketServer
+      super
+    end
+
+    def self.instance_name
+      "bitbucket_server"
```

Comment:
Would this support both Bitbucket and Bitbucket Server? If so then it probably should be just `bitbucket` ( for example we don't have different Dangerfile syntax for both GitHub and GitHub Enterprise ( or gitlab.com / self hosted GitLab )